webapp/apps/dynamic/models.py

```diff
@@ -87,6 +87,27 @@ class DynamicBehaviorSaveInputs(models.Model):
     micro_sim = models.ForeignKey(OutputUrl, blank=True, null=True,
                                   on_delete=models.SET_NULL)
 
+    def get_tax_result(self):
+        """
+        If taxcalc version is greater than or equal to 0.13.0, return table
+        If taxcalc version is less than 0.13.0, then rename keys to new names
+        and then return table
+        """
+        outputurl = OutputUrl.objects.get(unique_inputs__pk=self.pk)
+        taxcalc_vers = outputurl.taxcalc_vers
+        # only the older (pre 0.13.0) taxcalc versions are null
+        if taxcalc_vers:
+            taxcalc_vers = LooseVersion(taxcalc_vers)
+        else:
+            return rename_keys(self.tax_result, PRE_TC_0130_RES_MAP)
+
+        # older PB versions stored commit reference too
+        # e.g. taxcalc_vers = "0.9.0.d79abf"
+        if taxcalc_vers >= LooseVersion("0.13.0"):
```

Comment:
You could put this in a constant somewhere, like "taxcalc_min_compat_version" or something... Might be a bit clearer where this number comes from, so that people looking at it in the future are less confused.

Reply:
Done, thanks.